### DISCUSSION TITLE
error: could not get cloud url: could not load current project: project is missing a 'name' attribute

### DISCUSSION THREAD

**Discussion by ibrahim-shaaban-hussein** [pulumi/pulumi#9825] [Pulumi in Practice] [answered]
$ pulumi new 
warning: A new version of Pulumi is available. To upgrade from version '3.33.2' to '3.34.0', visit https://pulumi.com/docs/reference/install/ for manual instructions and release notes.
error: could not get cloud url: could not load current project: project is missing a 'name' attribute

Ibrahims-MacBook-Pro:quickstart ibrahims.hussein$ pulumi new aws-python
warning: A new version of Pulumi is available. To upgrade from version '3.33.2' to '3.34.0', visit https://pulumi.com/docs/reference/install/ for manual instructions and release notes.
error: could not get cloud url: could not load current project: project is missing a 'name' attribute
Ibrahims-MacBook-Pro:quickstart ibrahims.hussein$ 

MacOs Catalina version 10.15.5

**Comment by ibrahim-shaaban-hussein** [comment]
I tried with two Pulumi versions v3.33.2 & V3.34.0 
both i have same error while i tried to type below commands even with complete argument of new project with aws-python and aws-s3... and azure-typescript, ....
**pulumi new**

**Comment by Frassle** [comment, marked as answer]
This looks like an issue rather than a discussion. I've raised https://github.com/pulumi/pulumi/issues/9836 if you could follow up there.
